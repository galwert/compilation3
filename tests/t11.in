int func(int arg)
{
    return 1;
}

int func(int arg) // Should fail here (Symbol already declared)
{
    return 1;
}

void main()
{
    print("Finished!");
}