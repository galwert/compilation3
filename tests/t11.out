---end scope---
arg INT -1
line 6: identifier func is already defined
